src/js_types.hpp

```diff
@@ -565,13 +569,6 @@ inline typename T::Value Object<T>::create_from_optional_app_error(ContextType c
     return create_from_app_error(ctx, *error);
 }
 
-
-
-template<typename T>
-inline typename T::Value Value<T>::from_uuid(typename T::Context ctx, const UUID& value) {
-    throw std::runtime_error("'UUID' type support is not implemented yet");
-}
-
```

Comment:
Shouldn't this function be implemented?
On line 588 there is a runtime_error for Type::Uuid.  I think this is where `from_uuid` needs to be implemented.

Reply:
It's depending on environment, so resides in `src/jsc/jsc_value.hpp` & `src/node/node_value.hpp` (this was inserted as a placeholder).